.github/workflows/guix-lint-check.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         run: |
           guix lint -L $GITHUB_WORKSPACE --exclude=archival \
           vtk-slicer@9.2 \
+          vtkaddon-slicer@b5aa061 \
```

Comment:
We use the suffix `-slicer` to denote those libraries which were forked by the Slicer project. `vtkAddon` was not forked by Slicer, but originally created for Slicer (I believe). I think we should rename the package to `vtkaddon` instead of `vtkaddon-slicer`